internal/testblobs/slow.go

```diff
@@ -84,6 +84,12 @@ func (slow *SlowBlobs) Delete(ctx context.Context, ref storage.BlobRef) error {
 	return slow.blobs.Delete(ctx, ref)
 }
 
+// DeleteWithStorageFormat deletes the blob with the namespace, key, and format version
+func (slow *SlowBlobs) DeleteWithStorageFormat(ctx context.Context, ref storage.BlobRef, formatVer storage.FormatVersion) error {
+	slow.sleep()
```

Comment:
what is this `slow.sleep()` for? I'm sure there's a good reason since there's a function built specifically for this but it's not obvious to me.

Reply:
This is part of `testblobs`, it's a blobstore that is arbitrarily slow, for testing purposes. Compare with the other methods in the store.

Reply:
Ah! That makes more sense haha.